### ISSUE TITLE
Feature request: override specific violations

### ISSUE THREAD

**Issue by gadfort** [nmoroze/tclint#6]
Something like flake8 provides where violations can be overridden with `#  noqa E711` at the end of the line.

**Comment by nmoroze** [comment]
Thanks for the request - implemented in 60a44e4a0f4e2dde57fb2a58d035a8031c53963c and shipped in v0.1.3!

**Comment by nmoroze** [comment]
I chose a comment style similar to [ESLint](https://eslint.org/docs/latest/use/configure/rules#using-configuration-comments-1). Example:

```tcl
# tclint-disable spacing
puts  "illegal"
# tclint-enable spacing

# tclint-disable-next-line indent, spacing
  puts  "also illegal"

puts too many arguments ! ;# tclint-disable-line command-args

# tclint-disable
 puts  "illegal"
# tclint-enable
```
